src/hera_pipelines/cli.py

```diff
@@ -165,3 +166,107 @@ async def run_day_loop(days, stage_dir, root_stage, max_simultaneous_days):
 
     cns.print(f"[bold blue]Starting aynchronous loop over {len(days)} days with {max_simultaneous_days} simultaneous days")
     asyncio.run(run_day_loop(days, stage_dir, root_stage, max_simultaneous_days))
+
+
+@main.command()
+@click.argument("season", type=click.Choice(list(seasons.seasons.keys())))
+@click.argument("idr", type=int)
+@click.argument("generation", type=int)
+@click.argument("repodir", type=click.Path(exists=True, dir_okay=True, file_okay=False))
+@click.option("-f/-F", "--force/--no-force", help="overwrite existing")
+@click.option(
+    "--cases", multiple=True,
+    type=click.Tuple([click.Choice(['redavg', 'nonavg']), click.Choice(['abscal', 'smoothcal']), click.Choice(['dlyfilt', 'inpaint'])]),
+    help="triplet of [redavg/nonavg, abscal/smoothcal, dlyfilt/inpaint]"
+)
+@click.option("--setup-analysis/--only-repo", default=True, help="whether to also softlink output tomls to the analysis dir")
+@click.option('--prefix', type=str, default='', help='a prefix to add to the casenames')
+@click.option("--all-cases/--specify-cases")
+def lstbin_setup(season, idr, generation, repodir, cases, force, setup_analysis, prefix, all_cases):
+    """Setup lstbin TOML files for a range of cases for a specific SEASON, IDR, and GENERATION.
+
+    Example for SEASON is "h6c" (which is also the first season this script works for).
+    The IDR is specified as "IDR.GENERATION" (eg. IDR 2.1)
```

Comment:
to be clear, this is the TOML that controls the makeflow, not the LST Configurator TOML, right?

Reply:
Actually, it's both. The `[FILE_CFG]` section is read by `hera_opm` as the LST Configurator TOML (it could in principle stand alone if you were to create the configurator in your own script). The rest of the TOML controls other aspects of hera_opm (e.g. the `[LSTAVG_OPTS]`  section is what gets read by the notebook itself).

Reply:
Maybe clarify that in the text?